### PR TITLE
wire up emulators, start firestore stuff

### PR DESCRIPTION
**Pull request by stdavis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,3 +20,9 @@ The restore service consists of a website that allows you to view your backups a
 1. conda activate moonwalk-backup
 1. cd jobs/backup
 1. pip install -e ".[tests]"
+
+Enable versioning on the emulator bucket:
+
+```bash
+gcloud storage buckets update gs://ut-dts-agrc-moonwalk-dev.appspot.com --versioning
+```
```
Comment (steveoh) [inline README.md:28]: **question**: How does gcloud know about the emulators? Wouldn't this run on the cloud project?

Comment (stdavis) [reply README.md:28]: I'm not sure. I wonder if gcloud knows when the emulators are running. This did seem to work though. I have versioning working with the emulator.

**Review by jacobdadams — APPROVED** [review]
Looks good to me. I did some searching and it looks like they do some name management so that `arcgis.GIS` is the same reference as `arcgis.gis.GIS`, so I guess that's why that's working.

**Comment by stdavis** [comment]
I'm going to cowboy this PR and merge it so that I can work on the cloudrun deploy today. Thanks for the review, @jacobdadams

**Pull request merged**